core/ui/TagTemplate.tid

```diff
@@ -27,4 +33,4 @@ color:$(foregroundColor)$;
 </span>
 \end
 
-<$macrocall $name="tag-body" colour={{!!color}} palette={{$:/palette}}/>
```

Comment:
I still can't tell if I was able to take out the replace last line that I wasn't intending when I originally submitted this.